functions/MatchDate.ts

```diff
@@ -12,11 +12,10 @@ const MatchDate: MatchFunction<Props> = (
   props,
 ) => {
   const now = new Date();
-  const start = new Date(props.start);
-  const end = new Date(props.end);
-  const isMatch = now > start && now < end;
+  const start = props.start ? now > new Date(props.start) : true;
+  const end = props.end ? now < new Date(props.end) : true;
```

Comment:
Idk if others think like me and if it's make sense, but i prefer take a look in a interval comparison like this: `lower value < higher value`. Using this for both sentences `lower value < mid value < higher value`